polkadot/runtime/polkadot/src/lib.rs

```diff
@@ -215,7 +215,8 @@ impl pallet_scheduler::Config for Runtime {
 	type MaximumWeight = MaximumSchedulerWeight;
 	// The goal of having ScheduleOrigin include AuctionAdmin is to allow the auctions track of
 	// OpenGov to schedule periodic auctions.
-	type ScheduleOrigin = EitherOf<EnsureRoot<AccountId>, AuctionAdmin>;
+	// Also allow Treasurer to schedule recurring payments.
+	type ScheduleOrigin = EitherOf<EitherOf<EnsureRoot<AccountId>, AuctionAdmin>, Treasurer>;
```

Comment:
Seems more readable since root is "special" although perhaps the tracks should go into a tuple to avoid further `EitherOf` nesting?
```suggestion
	type ScheduleOrigin = EitherOf<EnsureRoot<AccountId>, EitherOf<Treasurer, AuctionAdmin>>;
```

Reply:
Agreed on tuple, this is much more readable:

```suggestion
	type ScheduleOrigin = (EnsureRoot<AccountId>, AuctionAdmin, Treasurer);
```

However, I just took a look and realized that `EnsureOrigin` doesn't have implementations on tuples, so we can't write this.